src/PerfView/EtwEventSource.cs

```diff
@@ -666,7 +673,8 @@ internal ETWEventRecord(ETWEventSource source, TraceEvent data, Dictionary<strin
             public DateTime OriginTimeStamp { get { return TimeZoneInfo.ConvertTime(LocalTimeStamp, this.m_source.OriginTimeZone); } }
             public override string Rest { get { return m_asText; } set { } }
             public EventIndex Index { get { return m_idx; } }
-
+            public override Payload[] Payloads { get { return m_payloads; }  }
```

Comment:
If you change `m_payloads` to be a `List<Payload>`, you can call `.ToArray()` here if an array is required for binding (not sure).  If not, you can keep it as a `List` and let it be enumerated by the GUI framework.